apps/nextjs-website/react-components/components/Footer/components/FooterColumn.tsx

```diff
@@ -0,0 +1,73 @@
+import { Link, Stack, Typography } from '@mui/material';
+import { hrefNoOp } from '../../../types/common/Common.types';
+import { LogoPagoPACompany } from '../assets/LogoPagoPACompany';
+import { EIcon } from '../../../components/common/EIcon';
+import { FooterColumnProps } from '../../..//types/Footer/Footer.types';
+import { Button } from '@mui/material';
+
+export const FooterColumn = ({
+  data,
+  companyLink,
+  icons,
+}: FooterColumnProps) => (
+  <Stack spacing={2} display='flex' alignItems={{ xs: 'center', sm: 'start' }}>
+    {data?.title && <Typography variant='overline'>{data.title}</Typography>}
+
+    {companyLink && (
+      <Button
+        role='link'
+        aria-label={companyLink.ariaLabel}
+        href={companyLink.href ?? hrefNoOp}
+        sx={{ display: 'inline-flex' }}
+      >
+        <LogoPagoPACompany />
+      </Button>
+    )}
+
+    {icons && (
+      <Stack
+        aria-label='i nostri social'
+        spacing={{ xs: 3, sm: 1.5, lg: 3 }}
+        direction='row'
+        component='ul'
+        alignItems={{ xs: 'center', sm: 'start' }}
+        sx={{ padding: 0, mt: 0.5, listStyle: 'none' }}
+      >
+        {icons?.map(
```

Comment:
```suggestion
        {icons.map(
```
`icons` is never undefined thanks to previous check.